bindings/ocaml/src/scheme.ml

```diff
@@ -0,0 +1,79 @@
+(* Generated by ocaml-rs *)
+
+open! Bigarray
+
+(* file: mod.rs *)
+
+type scheme = 
```

Comment:
Mapping scheme by hand seems not a good idea since it's hard for us to keep sync. How about just using string and refer to https://opendal.apache.org/docs/category/services?